softwareconfiguration/mlplan/sklearn/resources/automl/searchmodels/sklearn/sklearn-preprocessors.json

```diff
@@ -0,0 +1,32 @@
+{

```

Comment:
Why is there only a single preprocessor contained?

Reply:
Replaced search space to be up to date.